core/src/avm2/globals/flash/text/font.rs

```diff
@@ -103,23 +64,9 @@ pub fn has_glyphs<'gc>(
     this: Object<'gc>,
     args: &[Value<'gc>],
 ) -> Result<Value<'gc>, Error<'gc>> {
-    if let Some((movie, character_id)) = this.instance_of().and_then(|this| {
-        activation
-            .context
-            .library
-            .avm2_class_registry()
-            .class_symbol(this)
-    }) {
+    if let Some(font) = this.as_font() {
         let my_str = args.get_string(activation, 0)?;
-
-        if let Some(Character::Font(font)) = activation
-            .context
-            .library
-            .library_for_movie_mut(movie)
-            .character_by_id(character_id)
-        {
-            return Ok(font.has_glyphs_for_str(&my_str).into());
-        }
+        return Ok(font.has_glyphs_for_str(&my_str).into());
     }
 
     Ok(Value::Undefined)
```

Comment:
This would be better returning `false.into()` to be more clear about the return value (`hasGlyphs` returns a `bool`)

Reply:
Sure, it's existing code so I didn't consider it. Changing!